tosfs/core.py

```diff
@@ -413,6 +416,200 @@ def _try_dir_info(self, bucket: str, key: str, path: str, fullpath: str) -> dict
         except Exception as e:
             raise TosfsError(f"Tosfs failed with unknown error: {e}") from e
 
+    def exists(self, path: str, **kwargs: Union[str, bool, float, None]) -> bool:
```

Comment:
What does this `**kwargs: Union[str, bool, float, None]` mean ?  What's the value that people should pass ?

Reply:
Souds like it's not quite clear for me to construct the argumenst when i call this method....